docs/src/content/docs/es/manual-setup.mdx

```diff
@@ -125,8 +125,8 @@ En el futuro, planeamos soportar mejor este caso de uso para evitar la necesidad
 
 ### Usa Starlight con SSR
 
-Por el momento, Starlight no soporta el [despliegue SSR](https://docs.astro.build/es/guides/server-side-rendering/) usando los adaptadores de servidor de Astro. Esperamos poder soportar esto pronto.
+Para habilitar el SSR, sigue la guía [“Adaptadores de Renderizado bajo Demanda”](https://docs.astro.build/es/guides/server-side-rendering/) en la documentación de Astro para agregar un adaptador de servidor a tu proyecto Starlight.
 
 Puedes usar Starlight junto a páginas personalizadas renderizadas bajo demanda en tu proyecto siguiendo la guía [“Adaptadores de Renderizado bajo Demanda”](https://docs.astro.build/es/guides/server-side-rendering/) en la documentación de Astro.
```

Comment:
Looks like we somehow duplicated these two paragraphs
| English | Spanish |
|---|---|
| ![image](https://github.com/user-attachments/assets/7364ff1d-1e37-4296-b140-3104084407f3) | ![image](https://github.com/user-attachments/assets/2f8e9903-b91b-497a-b938-a5663f4cd120) |

I guess we want to delete the second one:

```suggestion
Para habilitar el SSR, sigue la guía [“Adaptadores de Renderizado bajo Demanda”](https://docs.astro.build/es/guides/server-side-rendering/) en la documentación de Astro para agregar un adaptador de servidor a tu proyecto Starlight.
```